src/routes/favoriteHoteles.js

```diff
@@ -0,0 +1,72 @@
+import { Router } from "express";
+import { param, query } from "express-validator";
+import { validateJwt } from "../middleware/validate-jwt.js";
+import { isAdminLogged } from "../middleware/is-logged.js";
+import { validateRequestParams } from "../middleware/validate-request-params.js";
+
+import {
+  favoritesGet,
+  getFavoriteById,
+  putFavorite,
+  favoriteDelete,
+  favoritePost,
+} from "../controller/favoriteHotel.controller.js";
+
+const router = Router();
+
+router.get(
+  "/",
+  [
+    query("limite")
+      .optional()
+      .isNumeric()
+      .withMessage("El límite debe ser un valor numérico"),
+    query("desde")
+      .optional()
+      .isNumeric()
+      .withMessage("El valor desde debe ser numérico"),
+    validateRequestParams,
```

Comment:
me gusta mucho que valides de forma opcional. pero, de nuevo, necesito que cambies `desde` por `page`

```suggestion
    query("limite")
      .optional()
      .isNumeric()
      .withMessage("El límite debe ser un valor numérico"),
    query("page")
      .optional()
      .isNumeric()
      .withMessage("El valor desde debe ser numérico"),
    validateRequestParams,
```

ademas deberias validar que los numeros no sean negativos. pero bueno, asi esta bien